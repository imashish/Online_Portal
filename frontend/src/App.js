import React from 'react';
import 'bootstrap/dist/css/bootstrap.min.css';
import './App.css';
import Header from './components/header';
import AppRouter from './router/AppRouter';
import Footer from './components/footer';

function App() {
	return (
		<React.Fragment>
			<Header />
			<main>
				<div className="container">
					<AppRouter />
				</div>
			</main>
			<Footer />
		</React.Fragment>
	);
}

export default App;
