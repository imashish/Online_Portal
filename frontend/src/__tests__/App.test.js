import App from '../App';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
	const component = renderer.create(
		<BrowserRouter>
			<App />
		</BrowserRouter>
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
