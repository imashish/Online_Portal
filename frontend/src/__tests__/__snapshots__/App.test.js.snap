// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <header>
    <nav
      className="navbar navbar-expand navbar-dark bg-dark"
    >
      <div
        className="container"
      >
        <a
          className="navbar-brand"
          href="/"
          onClick={[Function]}
        >
          A2Z SHOP
        </a>
        <div
          className="navbar-nav ml-auto"
        >
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="/login"
              onClick={[Function]}
            >
              Login
            </a>
          </li>
        </div>
      </div>
    </nav>
  </header>,
  <main>
    <div
      className="container"
    >
      <div
        className="container mt-3"
      >
        <div
          className="row pb-1"
        >
          <div
            className="input-group col-lg-4"
          >
            <h3>
              Welcome to the online shopping portal.
            </h3>
          </div>
        </div>
      </div>
    </div>
  </main>,
  <footer>
    <div
      className="container"
    >
      <div
        className="text-center py-3"
      >
        Copyright Â© A2Z Shop
      </div>
    </div>
  </footer>,
]
`;
